### ISSUE TITLE
Allow more ARIA attributes that are string values to be empty

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4108]
There are a few newer ARIA attributes of type: string that I think should be allowed as empty:

- aria-braillelabel
- aria-brailleroledescription
- aria-valuetext

Since both `aria-label` and `aria-roledescription` are allowed to be empty, I see no reason why their braille equivalents shouldn't be. As for valuetext, it seems fairly reasonable that values can be empty, so aria-valuetext should be allowed empty too.

**Comment by padmavemulapati** [comment]
validated using latest axe-core develop branch code base, 
`braille equivalent` rule also allowing 
aria-braillelabel
aria-brailleroledescription
aria-valuetext allowing empty



![Image](https://github.com/dequelabs/axe-core/assets/47409579/d6e908d3-6a06-4ecc-823b-1ff4a2be6dd8)

Recording tells us allowed and fail conditions of the checks:



https://github.com/dequelabs/axe-core/assets/47409579/c2472d9a-19d1-4e86-859d-017640f8996e
